.github/ISSUE_TEMPLATE/bug-report.md

```
@@ -1,24 +1,24 @@
 ---
 name: "🐛 Bug Report"
-about: 버그나 오류 제보
+about: Report a bug or issue
 title: "🐛 [Bug] "
 labels: bug
 assignees: ''
 ---
 
-## 🐞 버그 설명
-발생한 문제에 대해 간략히 설명해주세요.
+## 🐞 Bug Description
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)